File: WebApplication1/WebApplication1/Models/Saloon.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models;

public partial class Saloon
{
    public int SaloonId { get; set; }

    public string Name { get; set; } = null!;

    public virtual ICollection<Workshop> Workshops { get; set; } = new List<Workshop>();
}

File: Mobile/app/src/main/java/com/example/studentmobileapp/Models/Saloon.kt
package com.example.studentmobileapp.Models

import kotlinx.serialization.Serializable


@Serializable
public class Saloon {
  var saloonId: Int = 0
  var saloonName: String = ""
  var workshops: List<Workshop> = listOf()
}

@Serializable
public class GetSaloonResp {
  var status: Int = 0
  var message: String = ""
  var data: List<Saloon?> = listOf()
}
